packages/services/src/services/CloudAccountAuth/CloudAccountAuth.js

```diff
@@ -1,35 +1,33 @@
 /**
- * Copyright IBM Corp. 2021
+ * Copyright IBM Corp. 2021, 2022
  *
  * This source code is licensed under the Apache-2.0 license found in the
  * LICENSE file in the root directory of this source tree.
  */
 import axios from 'axios';
-import Cookies from 'js-cookie';
+import root from 'window-or-global';
 
 /**
```

Comment:
We can remove this cookie-related documentation.

Reply:
@huntermacd Where is this cookie-related documentation that you're referencing?

Reply:
maybe this part? https://github.com/carbon-design-system/carbon-for-ibm-dotcom/pull/8149/files#diff-f125cbf578fa155879790dfaa7d41016c75a0d15b871448d44cf53cc27b5a6ceR11

Reply:
Yep, those lines.